### PR TITLE
Add GraphQL context.set function to release note

### PR DESCRIPTION
**Pull request by ThisaruGuruge**
## Purpose
$Subject

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-beta6/RELEASE_NOTE.md [hunk 1]
```diff
@@ -89,6 +89,21 @@ Allowed listener-level interceptors to have only the default path.
 
 ### Breaking Changes
 
+#### `graphql` Package
+Deprecated the `add` function in the `graphql:Context` object and introduced the `set` function.
+
+Existing method:
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-beta6/RELEASE_NOTE.md:95]:
```suggestion
**Old Method:**
```

File: downloads/swan-lake-release-notes/swan-lake-beta6/RELEASE_NOTE.md [hunk 2]
```diff
@@ -89,6 +89,21 @@ Allowed listener-level interceptors to have only the default path.
...
+```ballerina
+graphql:Context context = new;
+graphql:Error? result = context.add("key", "<value>"); // Can return an error
+```
+
+New method:
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-beta6/RELEASE_NOTE.md:101]:
```suggestion
**New Method:**
```

File: downloads/swan-lake-release-notes/swan-lake-beta6/RELEASE_NOTE.md [hunk 3]
```diff
@@ -89,6 +89,21 @@ Allowed listener-level interceptors to have only the default path.
 
 ### Breaking Changes
 
+#### `graphql` Package
+Deprecated the `add` function in the `graphql:Context` object and introduced the `set` function.
+
+Existing method:
+```ballerina
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-beta6/RELEASE_NOTE.md:96]:
```suggestion
```ballerina
```
Please check the code formatting.

File: downloads/swan-lake-release-notes/swan-lake-beta6/RELEASE_NOTE.md [hunk 4]
```diff
@@ -89,6 +89,21 @@ Allowed listener-level interceptors to have only the default path.
...
+graphql:Context context = new;
+graphql:Error? result = context.add("key", "<value>"); // Can return an error
+```
+
+New method:
+```ballerina
+graphql:Context context = new;
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-beta6/RELEASE_NOTE.md:103]:
```suggestion
graphql:Context context = new;
```
Please check the code formatting.

File: downloads/swan-lake-release-notes/swan-lake-beta6/RELEASE_NOTE.md (hunk 3 above)
Comment (ThisaruGuruge) [reply downloads/swan-lake-release-notes/swan-lake-beta6/RELEASE_NOTE.md:96]: Looks fine for me.

**Review by praneesha — APPROVED** [review]

**Pull request merged**